script.js

```diff
@@ -0,0 +1,173 @@
+let movieRegistry = [];
```

Comment:
For global variables like this where they are important to the core functionality of the app, it can be helpful to add some context for readers. 
```suggestion
/**
 * List of all movies displayed to users
 */
let movieRegistry = [];
```